### DISCUSSION TITLE
Regression tests all failed

### DISCUSSION THREAD

**Discussion by Uisliu** [OpenFAST/openfast#1856] [Q&A]
Hello!

I've been building OpenFAST from source on windows Using Visual Studio 2019. Up until this point I have manged to get to a point where I can build everything with 0 errors(some warnings arise while building but disappear once it is built). My issue arises when performing regression tests. Im using Anaconda 3 when preforming the test, I enter the commands as prompted in the OpenFAST documentation 3.2.4.4.1. However once I conduct the Regresssion test I get the following error for every module.

![image](https://github.com/OpenFAST/openfast/assets/71447919/e429cc39-28e5-410a-8991-58429775b6db)

this happens for all 40 modules tested and they all fail :(

I would appreciate some help and I am willing to give all info.
something else to note is that when I was When I entered the command to download the testing data base, there were some issues as well.
![image](https://github.com/OpenFAST/openfast/assets/71447919/525920c7-4f10-45d6-8b4b-f52a8a577bdf)

this was the expected outlet from the command prompt

![image](https://github.com/OpenFAST/openfast/assets/71447919/7756f79a-9d46-4532-9a1a-44cc243a4d08)

and this was the outlet I got

**Comment by andrew-platt** [comment]
Greetings!

The git submodule commit for r-test, `1cbf6a1ae96655e2a7fa2a6865ccc99fc39bf6c8`, corresponds to the `dev` branch.  Unless you are developing code, you probably want to use the `main` branch of both OpenFAST and the r-test repositories.  If you have trouble with the `main` branches as well, please let us know soon so that we can fix it before our 3.5.2 release.

The `dev` branch is the development branch and may not be fully functional at the moment with regression testing on Windows (there were some changes to the testing methodology that may not be fully accounted for on the Windows platform -- this is a bit concerning to me as I thought this had been at least partially updated).

Regards,
